tests/cnf/core/network/metallb/tests/frrk8-tests.go

```diff
@@ -276,6 +233,10 @@ var _ = Describe("FRR", Ordered, Label(tsparams.LabelFRRTestCases), ContinueOnFa
 				err = metallbenv.CreateNewMetalLbDaemonSetAndWaitUntilItsRunning(tsparams.DefaultTimeout, workerLabelMap)
 				Expect(err).ToNot(HaveOccurred(), "Failed to recreate metalLb daemonset")
 
+				By("Verifying that the frrk8sPod deployment is in Ready state and create a list of the pods on " +
+					"worker nodes.")
+				frrk8sPods = verifyAndCreateFRRk8sPodList()
+
```

Comment:
I beleive you can remove it

Reply:
I put it there in case the test case is run by itself ie FIt